poc/vidpf.py

```diff
@@ -102,6 +110,7 @@ def eval(cls, agg_id, correction_words, init_seed, level, prefixes, cs_proofs, p
         #
         # Implementation note: We can save computation by storing
         # `prefix_tree_share` across `eval()` calls for the same report.
+        pi_proof = cls.ROOT_PROOF
```

Comment:
The reason that I was passing the proof as an argument here is that we can take the previous proof and update it. Here we're only reading the ROOT_PROOF but we're not updating anywhere, right? Or I missed it?

Reply:
I believe so; the ROOT_PROOF is just a constant taking the place of the empty string in the original PLASMA design.

Reply:
That's right, but for Mastic we weren't updating the proof either, so I expect this to be a non-breaking change.

If we think that starting evaluation with an arbitrary proof would be useful, then we can add this back to the API when we need it.